docs/UserGuide.md

```diff
@@ -145,7 +145,7 @@ Examples:
 
 ### Adding an insurance plan to a client : `addInsurance`
 
-Adds the insurance plan (given by the `INSURANCE_ID`) to the client at the specified `INDEX`.
+Adds the insurance plan (given by the `INSURANCE_ID`) to the client at the specified `INDEX`. Insurance plans of different types cannot be added by the user and will be added beforehand to ensure that the plans are consistent with market type plans.
```

Comment:
Could you clarify what "Insurance plans of different types cannot be added by the user" means?

Reply:
I think what he meant is that the insurance plans are pre-loaded and not added by the user. But yep, could be slightly clearer.

Maybe: "Note: Insurance plans are pre-loaded in the app to ensure consistency with market type plans."

Reply:
like if the user wants to add a new type of insurance say "Fire Insurance" they cant, cause that feature hasnt been implemented yet

Reply:
ok completed to make it clearer.